dark/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctYPE html>

```

Comment:
Can you undo these changes? @maddy020 
```suggestion
<!DOCTYPE html>
```